spec/system/casework/viewing_an_application/that_is_open_and_unassigned_spec.rb

```diff
@@ -21,6 +21,10 @@
     expect(page).to have_content('Copy reference number')
   end
 
+  it 'includes the urn copy link' do
+    expect(page).to have_content('Copy')
```

Comment:
This text needs to be updated also to match the link text. Same as spec below.

Reply:
This test is also failing @MohammedHassanLatif - the json fixture that this test runs against doesn't have a URN by the look of things. So this could actually be a good test to add - that the copy URN buttons does not show if there is not URN. 

Then you would need a new context block inside which we add a URN in the same way as [line 53 of this test ](https://github.com/ministryofjustice/laa-review-criminal-legal-aid/blob/5b56fcfaee5cf57e3469e979056b8500fd65c04d/spec/system/casework/viewing_an_application/that_is_open_and_unassigned_spec.rb#L53C7-L53C10)

then check that the link is there